app/javascript/application.js

```diff
@@ -0,0 +1,16 @@
+// Entry point for the build script in your package.json
+// import "@hotwired/turbo-rails"
+// import "./controllers"
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import store from './redux/configureStore';
+import { Provider } from 'react-redux'
+import { App } from './components/App';
+
+ReactDOM.render(
+  <Provider store={store}>
+    <App />
+  </Provider>,
+  document.getElementById('root'),
+);
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/235439644-6f21f0ed-fc64-47f5-b069-ba59e059593b.png)

- [ ] Your backend is working as expected 👍 but i guess your frontend part is having some issues 😨 Kindly fix the issue which you can see in the image above ⬆️ to make your frontend working as expected